### ISSUE TITLE
Changes of Pictures regarding the Setting "create parallel"

### ISSUE THREAD

**Issue by Toaster143** [cmdr2/stable-diffusion-ui#339]
It seems that Pictures changes at the same seed, same steps and same guidance if the setting  "Generate in parallel" is set to 2 (?)

**To Reproduce**
Choose any fixed Seed and prompt.
set "amount of pictures" to 2 and "parallel" to 1

create pictures

set "amounts of pictures" to 2 and "parallel" to 2

create pictures

**Expected behavior**
Usually the Pictures with the same seed and guide should be staying the same since it's seed, steps and guide didn't change.
Because this is not the case it can only mean, that the "parallel processing" somehow alters anything that I can't figure out.

I'm very sorry if this isn't a bug but it's an issue because those altered pictures can't be recreated by seed, guide and steps.

Thanks,
Sincerely

**Comment by MrManny** [comment]
Related to (and possibly a duplicate of) #311, #337.

**Comment by Winterhuman** [comment]
I'm not able to reproduce this, here's what I'm doing:

```
Prompt: a photograph of an astronaut riding a horse
Seed: 55555
Model: sd-v1-4
Sampler: plms
Size: 256x256
Interference: 25
Guidance: 7.5
```

```
Prompt: a photograph of an astronaut riding a donkey
Seed: 55555
Model: sd-v1-4
Sampler: plms
Size: 512x512
Interference: 50
Guidance: 5
```

I'm getting the same pair of images for both `Number: 2, Parallel: 1` and `Number: 2, Parallel 2`, the issues linked by @MrManny are probably the cause

**Comment by Toaster143** [comment]
I forgot to mention that I was using "Euler_a" as a sampler:

![image](https://user-images.githubusercontent.com/1832313/196268481-2ba65045-37c7-4615-8904-6fb3342fde16.png)

**Comment by Toaster143** [comment]
In plms I this Issue didn't occur at my machine either

**Comment by cmdr2** [comment]
Hi, is this still an issue? We released a few bug fixes related to this over the past 2 weeks. Thanks!

**Comment by MrManny** [comment]
> Hi, is this still an issue? We released a few bug fixes related to this over the past 2 weeks. Thanks!

Yes, this is still the case for at least `euler_a` and `dpm2_a`. This may be an undocumented characteristic of the ancestral samplers and not necessarily a bug. This is not a huge problem outside of #337. The easiest workaround is to limited batches for these two samplers to 1.

**Comment by Toaster143** [comment]
Since this seems to be due to the sampler I will mark this thread as closed. Thanks for your help!

**Comment by ChrisAcrobat** [comment]
I was just about to bug report this also. This did not come up when I was searching for "Draw another 25 steps". 👈 I'm hoping that GitHub's search engine will now catch the phrase.
